### PR TITLE
Issue #1026 Name ehcache cluster connections

### PR DESCRIPTION
**Pull request by albinsuresh**

### CODE REVIEW

File: clustered/client/src/main/java/org/ehcache/clustered/client/service/DefaultClusteringService.java [hunk 1]
```diff
@@ -86,7 +86,9 @@ public ClusteringServiceConfiguration getConfiguration() {
   @Override
   public void start(final ServiceProvider<Service> serviceProvider) {
     try {
-      clusterConnection = ConnectionFactory.connect(clusterUri, new Properties());
+      Properties properties = new Properties();
+      properties.put("connection.name", "EHCACHE:" + this.entityIdentifier);
```
Comment (ljacomet) [inline clustered/client/src/main/java/org/ehcache/clustered/client/service/DefaultClusteringService.java]: What is the entityIdentifier looking like? the whole URI? only the name part?
Also not sure I like the all caps for Ehcache.

Comment (chrisdennis) [inline clustered/client/src/main/java/org/ehcache/clustered/client/service/DefaultClusteringService.java]: The entityIdentifier is the just the "name" part.

Comment (lorban) [inline clustered/client/src/main/java/org/ehcache/clustered/client/service/DefaultClusteringService.java]: I would *at the very least* use constants for the "EHCACHE" identifier as well as the ":" separator.

**Comment by albinsuresh** [comment]
Addressed the review feedback and added a minimal unit test.

**Comment by ljacomet** [comment]
Good to merge once rebased

**Comment by albinsuresh** [comment]
test this please

**Pull request merged**